docs/release-notes/self-hosted/self-hosted-v3.5.0.md

```diff
@@ -0,0 +1,68 @@
+# Self-hosted v3.5.0
+
+These release notes are for [Codacy Self-hosted v3.5.0](https://github.com/codacy/chart/releases/tag/3.5.0){: target="_blank"}, released on March 31, 2021.
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+## Product enhancements
+
+-   Now, Codacy allows [configuring who is allowed to manage patterns and ignore files](https://docs.codacy.com/organizations/roles-and-permissions-for-synced-organizations/#change-analysis-configuration) on all supported Git providers. (CY-3883)
+-   Now, Codacy has [Checkov support](https://github.com/bridgecrewio/checkov), extending analysis to infrastructure-as-code like Terraform, Cloudformation and Kubernetes  (CY-3830)
```

Comment:
I want to propose that we rephrase this to highlight the tools/platforms we now support. Something along the lines of:
Codacy is now running Checkov(..), which extends our analysis for infrastructure-as-code: Terraform, Terraform Plan, Cloudformation, Kubernetes, Serverless framework, and ARM Templates are now supported. This new type of analysis contains more than 500 rules, covering compliance and security best practices for AWS, Azure and Google Cloud.

Reply:
Alright, sounds good to me!